app/assets/javascripts/winners/previousWinners.js

```diff
@@ -6,7 +6,9 @@
   window.winners = {
     onReady: function(){
       // Retrieve data
-      $.getJSON('/auctions.json').success(function(data){
+      var MP = 'https://micropurchase.18f.gov/auctions.json'
```

Comment:
also, that data is on prod now but was removed at https://github.com/18F/micropurchase/commit/32bafd1131a47f4a5c773d3a7508e27dc6ed9d88 so it won't be on prod once we do our next deploy -- can we use the actual API endpoint instead?